projects/frontend/shared-components/gui/projects/shared/src/lib/commons/ngx-components/search/search.component.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, ElementRef, EventEmitter, Input, Output, ViewChild } from "@angular/core";
+import { FormControl } from "@angular/forms";
+
+import { BehaviorSubject, combineLatest, Subject, Subscription } from "rxjs";
+import { debounceTime } from "rxjs/operators";
+
+const defaultSearchInputPadding = 24;
+
+@Component({
+    selector: "vdk-search",
+    templateUrl: "./search.component.html",
+    styleUrls: ["./search.component.scss"],
+})
+export class VmwSearchComponent {
```

Comment:
also VmwSearchComponent can be VdkSearchComponent. 

In general, check all the class names exported by the project and rename all of them.